cellprofiler/modules/graytocolor.py

```diff
@@ -182,25 +201,23 @@ def add_stack_channel_cb(self, can_remove=True):
             len(self.stack_channels) % len(DEFAULT_COLORS)]
         group.append("image_name", cps.ImageNameSubscriber(
                 "Image name", cps.NONE,
-                doc=
-                """
-                <i>Used only if %(SCHEME_STACK)s or %(SCHEME_COMPOSITE)s is
-                chosen</i><br>
-                Select the input image to add to the stacked image""" % globals()))
+                doc='''\
+*Used only if %(SCHEME_STACK)s or %(SCHEME_COMPOSITE)s is chosen*
```

Comment:
Add parentheses around this first sentence.

Add a blank line between *(Used only if...)* and the next line.